themes/wds_headless/inc/yoast-seo.php

```diff
@@ -19,17 +19,12 @@
 	 * @return array              Filtered breadcrumbs.
 	 */
 	function wds_breadcrumb_links( array $breadcrumbs ) {
-		if ( ! defined( 'HEADLESS_FRONTEND_URL' ) ) {
-			return $breadcrumbs;
-		}
 
-		$base_url = rtrim( HEADLESS_FRONTEND_URL, '/' );
-
-		// Override domain in breadcrumbs.
-		return array_map( function( $breadcrumb ) use ( $base_url ) {
+		// Return path in breadcrumbs.
+		return array_map( function( $breadcrumb ) {
 			$parsed_url        = wp_parse_url( $breadcrumb['url'] );
 			$path              = $parsed_url['path'] ?? '';
-			$breadcrumb['url'] = "{$base_url}{$path}";
+			$breadcrumb['url'] = $path;
```

Comment:
I think the assumption was the `$base_url `will always contain `/`. With this change, aren't we going to end up with `data` instead of `/data` (and empty for the home page instead of `/` again)?

Reply:
@kilemensi you're right. I will add a preceding `/`